README.md

```diff
@@ -125,21 +91,66 @@ In order to develop for Grüvee you are going to need to have a few things insta
     - Android Build Tools 28.0.3 (Installable from Studio SDK Manager)
     - Android 9 SDK (Installable from Studio SDK Manager)
 
-## TODO - Add setup for .env file
+## Things Left To Document
 
-## TODO - Add setup for integrating gruveebackend repo
+-   Add setup for integrating gruveebackend repo
+-   Add setup for running Android emulator
+-   Android startup `emulator -avd Pixel_2_API_29`
+-   Add APK build process
+-   Add Assembly process for APK `./gradlew app:assembleRelease`
 
-## TODO - Add setup for running Android emulator
+# Running Grüvee Locally
 
--   `emulator -avd Pixel_2_API_29`
+Now we are at the GOOD stuff. Time to get Grüvee running locally. Make sure you have the [prerequisites](#prerequisites) installed. Open up **two instances** of your favorite terminal/command prompt and navigate to the root directory of the Grüvee repository for both of them. One is going to run your React Native bundle server and the other will start the Grüvee iOS or Android app.
 
-## TODO - Add APK build process
+## 🌲 Environment Variables
 
--   `./gradlew app:assembleRelease`
+We have included an [`.example.env`](/.example.env) that has the properties that are needed for the app to run properly. Start here by making a `.env` in the root of the repo
 
-# Running Grüvee Locally
+### `ENVIRONMENT`
```

Comment:
I think you could probably put the environmental variable directions inside the `README-Support/` section.